packages/grafana-ui/src/components/Icon/cached.json

```diff
@@ -158,6 +158,7 @@
   "custom/gf-interpolation-step-after",
   "custom/gf-interpolation-step-before",
   "custom/gf-logs",
+  "custom/gf-ml",
```

Comment:
In general terms icons should only be added to this list if they will appear on first page load. The problem with the icon is not about not being in this list but was having the incorrect name in the `icon.ts` type list.

So if this icon is not directly display on first page load it is better to remove it from here and only keep it inside icon.ts. Adding here will only increase the size of the JS bundle without good reason.